src/Flowdock/FlowdockHandler.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MonologHttp\Flowdock;
+
+use Monolog\Formatter\FormatterInterface;
+use Monolog\Logger;
+use MonologHttp\AbstractHttpClientHandler;
+use MonologHttp\Flowdock\Formatter\FlowdockFormatterInterface;
+use MonologHttp\Flowdock\Formatter\FlowdockMessageFormatter;
+use Psr\Http\Client\ClientInterface;
+use Psr\Http\Message\RequestFactoryInterface;
+use Psr\Http\Message\RequestInterface;
+use Psr\Http\Message\UriInterface;
+use Monolog\Handler\HandlerInterface;
+
+final class FlowdockHandler extends AbstractHttpClientHandler
+{
+
+    /**
+     * @var UriInterface|string
+     */
+    private $uri;
+
+    /**
+     * @param string|UriInterface $uri
+     * @param int|string $level
+     */
+    public function __construct(
+        ClientInterface $client,
+        RequestFactoryInterface $requestFactory,
+        $uri,
+        $level = Logger::ERROR,
+        bool $bubble = true
+    ) {
+        parent::__construct($client, $requestFactory, $level, $bubble);
+        $this->uri = $uri;
+    }
+
+    protected function createRequest(array $record): RequestInterface
+    {
+        $body = \json_encode($record['formatted']);
+        if (\JSON_ERROR_NONE !== \json_last_error()) {
+            throw new \InvalidArgumentException('Could not format record to json: ' . \json_last_error_msg());
+        }
+
+        $request = $this->requestFactory->createRequest('POST', $this->uri)
+            ->withHeader('Content-Type', ['application/json']);
```

Comment:
Are you able to send without an authentication?

Reply:
The token can be parsed in the `uri` also 
I think is more simple to provide just one way either parse it in constructor/and the header or accept it through uri

I want to try it to see how it works exactly (since I've never used the app) and may write a readme with usage

Reply:
So I checked it and the simplest and recommended way to send log messages is to create a source and parse a `flow_token` in the url

https://www.flowdock.com/api/integration-getting-started